src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionBase.cs

```diff
@@ -180,6 +190,32 @@ public Task StopAsync()
             return Task.CompletedTask;
         }
 
+        public async Task CloseAsync()
+        {
+            if (!_closeConnectionLock.Wait(0))

```

Comment:
We never call release?

Reply:
We don't need to, The `closeAsync` will be called only once.